content/intro-to-storybook/vue/es/using-addons.md

```diff
@@ -0,0 +1,213 @@
+
+Primero, importe el decorador `withKnobs` y el tipo de knob `object` a `Task.stories.js`:
+
+```javascript
+// src/components/Task.stories.js
+
+import { storiesOf } from '@storybook/vue';
+import { action } from '@storybook/addon-actions';
+import { withKnobs, object } from '@storybook/addon-knobs';
+```
+
+A continuación, dentro de las historias de `Task`, pase`withKnobs` como parámetro a la función `addDecorator()`:
+
+```javascript
+// src/components/Task.stories.js
+
+storiesOf('Task', module)
+  .addDecorator(withKnobs)
+  .add(/*...*/);
+```
+
+Por último, integre el tipo de knob `object` dentro de la historia "predeterminada":
+
+```javascript
+// src/components/Task.stories.js
+storiesOf('Task', module)
+  .addDecorator(withKnobs)
+  .add('default', () => {
+    return {
+      components: { Task },
+      template: `<task :task="task" @archiveTask="onArchiveTask" @pinTask="onPinTask"/>`,
+      props: {
+        task: {
+          type: Object,
+          default: object('task', { ...task }),
+        },
+      },
+      methods,
+    };
+  })
+  .add('pinned', () => {
+    return {
+      components: { Task },
+      template: `<task :task="task" @archiveTask="onArchiveTask" @pinTask="onPinTask"/>`,
+      data: () => ({ task: { ...task, state: 'TASK_PINNED' } }),
+      methods,
+    };
+  })
+  .add('archived', () => {
+    return {
+      components: { Task },
+      template: `<task :task="task" @archiveTask="onArchiveTask" @pinTask="onPinTask"/>`,
+      data: () => ({ task: { ...task, state: 'TASK_ARCHIVED' } }),
+      methods,
+    };
+  });
+```
+
+Ahora debería aparecer una nueva pestaña "Knobs" al lado de la pestaña "Action Logger" en el panel inferior.
+
+Como se documenta [here](https://github.com/storybooks/storybook/tree/master/addons/knobs#object), el tipo `object` del knob acepta una etiqueta y un objeto predeterminado como parámetros. La etiqueta es constante y se muestra a la izquierda de un campo de texto en el panel de complementos. El objeto que ha pasado se representará como un blob JSON editable. ¡Siempre que envíe un JSON válido, su componente se ajustará en función de los datos que se pasan al objeto!
+
+## Complementos evolucionan el alcance de tus Storybooks
+
+Su instancia de Storybook no solo sirve como un maravilloso [CDD environment](https://blog.hichroma.com/component-driven-development-ce1109d56c8e), sino que ahora estamos proporcionando una fuente interactiva de documentación. Los PropTypes son geniales, pero un diseñador o alguien completamente nuevo en el código de un componente podrá descubrir su comportamiento muy rápidamente a través de Storybook con el complemento de knobs implementado.
+
+## Usando Knobs para encontrar casos de borde
+
+Además, con un fácil acceso para editar los datos pasados ​​a un componente, los QA Engineers o los UI Engineers ahora pueden llevar un componente al límite. Como ejemplo, ¿qué le sucede a `Task` si nuestro elemento de la lista tiene un string _MASSIVE_?
```

Comment:
update "_MASSIVE_" to "_GIGANTESCA_", if you don't mind, it's a valid translation and avoids mixing in languages.